instrumentation/src/javatest/org/oppia/android/instrumentation/EndToEndTestHelper.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.android.instrumentation
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import org.junit.Assert.assertNotNull
+
+/** This object contains common operations used for end-to-end tests. */
+object EndToEndTestHelper {
```

Comment:
Can we go with the other approach where we can make `EndToEndTestHelper` a class and put the function in companion object making them static to access.